DEVELOPMENT.md

```diff
@@ -1,30 +1,50 @@
-# How do we work?
+# Developer Guide
 
-Our workflow is to put changes in feature branches which we submit for review
-on GitHub as pull requests. Ideally a pull request is small and changes only
-one aspect of the code at a time. After a pull request is reviewed by at least
-one peer and passes all tests, it can be squash-merged into master.
+## Development prerequisites
+
+* [NodeJS](https://nodejs.org/en/)
+* [Yarn](https://yarnpkg.com/getting-started/install)
+* [Rustup](https://github.com/rust-lang/rustup)
+* Dependencies of [Cypress](https://docs.cypress.io/guides/getting-started/installing-cypress#System-requirements)
 
-💡 *We require all commits to be signed for a branch to be merged into
-master. Learn more on setting up [commit signing][cs].*
+## Merging changes into master
 
-To automate our release process as much as possible we're using
-[Standard Version][sv]. Commits on master should be formatted according to
-the [conventional commits specification][cc].
+Maintainers are responsible for adding contribution to the `master` branch on
+the command line. We avoid the Github UI for merging changes.
```

Comment:
We could link to the explanation: https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0003.md#merging-pull-requests.

Reply:
Done